### PR TITLE
#92: Add support for other languages.

### PR DESCRIPTION
**Pull request by 2Fast2BCn**
Add support for other languages.

### CODE REVIEW

File: database/migrations/000041_add_language_to_settings.down.psql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE settings DROP COLUMN default_language;
```
Comment (maxwedwards) [inline database/migrations/000041_add_language_to_settings.down.psql:1]: We don't bother with down migrations as for safety we will never run a down migration

Comment (maxwedwards) [reply database/migrations/000041_add_language_to_settings.down.psql:1]: So we just leave the file blank

File: web/src/App.test.tsx [hunk 2]
```diff
@@ -13,5 +13,9 @@ test("renders login page", () => {
     </BrowserRouter>
   );
 
-  expect(getByText(/login/i)).toBeInTheDocument();
+  expect(getByText(/loading/i)).toBeInTheDocument();
+
+  waitFor(() => {
+    expect(getByText(/login/i)).toBeInTheDocument()
+  });
```
Comment (maxwedwards) [inline web/src/App.test.tsx:20]: Any need for the waitfor? That's the default behaviour anyway.

File: cmd/torq/internal/torqsrv/server.go [hunk 3]
```diff
@@ -194,6 +194,15 @@ func registerStaticRoutes(r *gin.Engine) {
 			c.File("./web/build/static/" + parts[len(parts)-2] + "/" + parts[len(parts)-1])
 			return
 		}
+
+		// locales json files
+		if strings.Contains(path, "/locales/") && strings.Contains(path, ".") &&
+			(strings.Contains(path, "json")) {
```
Comment (maxwedwards) [inline cmd/torq/internal/torqsrv/server.go:200]: Combine the "." and the "json" to be ".json"?

File: web/src/App.test.tsx (hunk 2 above)
Comment (2Fast2BCn) [reply web/src/App.test.tsx:20]: I tried but it does not work without it :-/

**Pull request merged**